src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/Symbols/Attributes/DataTypeAttributeSymbols.cs

```diff
@@ -2,13 +2,43 @@
 
 using Microsoft.CodeAnalysis;
 using Acuminator.Utilities.Roslyn.Constants;
+using System.Collections.Immutable;
+using System.Collections.Generic;
 
 namespace Acuminator.Utilities.Roslyn.Semantic.Symbols
 {
 	public class DataTypeAttributeSymbols : SymbolsSetBase
 	{
 		internal DataTypeAttributeSymbols(Compilation compilation) : base(compilation)
-		{ }
+		{
+			var types = new List<INamedTypeSymbol?>() {
+				PXDBStringAttribute,
+				PXStringAttribute,
+				PXDBBinaryAttribute,
+				PXDBDecimalAttribute,
+				PXDecimalAttribute,
+				PXDBDoubleAttribute,
+				PXDoubleAttribute,
+				PXDBFloatAttribute,
+				PXFloatAttribute,
+			};
+
+			DataAttributesWithLength = types.ToImmutableArray()!;
+
+			DataAttributesWithHardcodedLength = new Dictionary<INamedTypeSymbol, int>(SymbolEqualityComparer.Default)
```

Comment:
@acasj you can't just erase null for every retrieved attribute type symbol and add them. 
Some of these symbols are null in older Acumatica versions. Thus, each of this symbols has to be checked for null.

Reply:
done, local run: 841 test passed, 10 skipped